Dynavity/Dynavity/view/CanvasElementMapView.swift

```diff
@@ -18,22 +12,7 @@ struct CanvasElementMapView: View {
     }
 
     private func transformToView(element: CanvasElementProtocol) -> some View {
```

Comment:
Do you think we should remove this function and directly use the view factory in the foreach?

Reply:
Sounds good! Thanks